app/services/cached_subheading_service.rb

```diff
@@ -0,0 +1,74 @@
+class CachedSubheadingService
+  TTL = 24.hours
+
+  DEFAULT_INCLUDES = [
+    :section,
+    :heading,
+    :chapter,
+    'chapter.guides',
+    :footnotes,
+    :commodities,
+    'commodities.overview_measures',
+    'commodities.overview_measures.duty_expression',
+    'commodities.overview_measures.measure_type',
+  ].freeze
+
+  def initialize(subheading, actual_date)
+    @subheading = subheading
+    @actual_date = actual_date
+  end
+
+  def call
+    Rails.cache.fetch(cache_key, expires_in: TTL) do
+      Api::V2::Subheadings::SubheadingSerializer.new(presented_subheading, options).serializable_hash
+    end
+  end
+
+  private
+
+  def presented_subheading
+    @presented_subheading ||= begin
```

Comment:
I think we don't need to memoize here.
this is called only by `call` and it is inside a Rails.cache.fetch(...)
so external caching will be used instead of the memorization.